### ISSUE TITLE
img alt="" is detected as an error

### ISSUE THREAD

**Issue by dd8** [MicrosoftEdge/static-code-scan#117]
A missing ALT is always an error, but ALT="" (also called null ALT) is used to hide decorative images from screen readers:

https://www.w3.org/TR/WCAG20-TECHS/H67.html

There are some examples of decorative images here:

https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by molant** [comment]
This project is deprecated and no longer maintained. Please check [sonar](https://github.com/sonarwhal/sonar) if you need a similar tool but more flexible and modern.
